ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-LEAF1A.cfg

```diff
@@ -115,6 +115,18 @@ interface Ethernet9
    switchport mode trunk phone
    switchport
 !
+interface Ethernet10
+   no shutdown
+   no switchport
+!
+interface Ethernet10.10
+   description test mtu not rendered for VEOS platform
+   no shutdown
+   mtu 9000
```

Comment:
The purpose of the test was to check that the platform did _not_ update this. But since we removed the update of default platform settings, this is no longer relevant. Let's revert the changes to this test, since you cover it in the platform_settings tests.